pipelines/broad/arrays/single_sample/Arrays.wdl

```diff
@@ -304,8 +304,8 @@ workflow Arrays {
   }
 
   output {
-    String chip_well_barcode_output = IlluminaGenotypingArray.chip_well_barcode_output
-    Int analysis_version_number_output = IlluminaGenotypingArray.analysis_version_number_output
+    String ChipWellBarcodeOutput = IlluminaGenotypingArray.chip_well_barcode_output
```

Comment:
Unfortunately, this change requires an update to the pipeline version and the changelog

Reply:
Oh boo.  I had hoped it wouldn't but it certainly makes sense that it does!